includes/settings/settings.php

```diff
@@ -83,10 +87,26 @@ public function register_admin_menu() {
 			esc_html__( 'Elementor', 'elementor' ),
 			'manage_options',
 			self::PAGE_ID,
-			[ $this, 'display_settings_page' ],
+			[
+				$this,
+				$this->home_module->is_experiment_active() ? 'display_home_screen' : 'display_settings_page',
+			],
 			'',
 			'58.5'
 		);
+
+		if ( $this->home_module->is_experiment_active() ) {
+			add_action( 'elementor/admin/menu/register', function( Admin_Menu_Manager $admin_menu ) {
+				$admin_menu->register( 'settings', new Admin_Menu_Item( $this ) );
```

Comment:
Why `settings` as the Page ID?

Reply:
In the current setup, the settings page uses `elementor` as the Page ID, but when the experiment is active, the settings page should get its own Page Id, and the top level 'elementor' Page ID, will be used for the new home screen.